dispatch/dispatch.go

```diff
@@ -133,6 +138,134 @@ func (d *Dispatcher) run(it provider.AlertIterator) {
 	}
 }
 
+// AlertGroup represents how alerts alerts exist within an aggrGroup
+type AlertGroup struct {
+	Alerts   []*EnrichedAlert `json:"alerts"`
+	Labels   model.LabelSet   `json:"labels,omitempty"`
+	Receiver string           `json:"receiver,omitempty"`
+}
+
+// EnrichedAlert is the representation of an alert, with status info and
+// fingerprint.
+type EnrichedAlert struct {
```

Comment:
Since this is also being used in the apiv2 package, maybe it should be moved to `types`?